src/meta/MetaHttpHandler.cpp

```diff
@@ -139,5 +196,66 @@ std::string MetaHttpHandler::toStr(folly::dynamic& vals) const {
     return ss.str();
 }
 
+bool MetaHttpHandler::dispatchSSTFiles(const std::string& url,
+                                       int port,
+                                       const std::string& path,
+                                       const std::string& local) {
+    auto command = folly::stringPrintf("hdfs dfs -ls hdfs://%s:%d%s ",
+                                       url.c_str(), port, path.c_str());
+    LOG(INFO) << "List SST Files " << command;
```

Comment:
Whenever you are going to log out some infos, please try to let yourself understand them correctly without any confusion. **AT LEAST**, try to use correct grammar, just try. 

As for this log, I, personally, will be much more comfortable if you put a `: ` after `Files`.